ion_phys/tests/test_tss.py

```diff
@@ -14,6 +14,27 @@ def _steady_state_population(intensity):
 
 class TestTSS(unittest.TestCase):
     """two-state system tests"""
+    def test_spont_rates(self):
+        """Test the spontaneous rates satisfy relations in net rates"""
+        intensity_list = [1e-3, 1e-1, 0.3, 1, 1.0, 2, 10., 1.2e4]
```

Comment:
Why `intensity_list`? Intensity doesn't come into the spontaneous matrix.

Reply:
Exactly. This test veriefies the spontaneous matrix elements have the correct relation to the overal transition rates elements. This is verified at a range of intensities.

I'll improve documenation.